### DISCUSSION TITLE
Add a filter for live/published in the Entry Manager

### DISCUSSION THREAD

**Discussion by murtaugh** [ExpressionEngine/ExpressionEngine#2977] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
I need to see entries that are live on a site and not waiting for their entry date to hit.

**Comment by intoeetive** [comment]
@murtaugh can you add a bit more explanation to this? Like, what would should be dropdown options and what be the entries list for each of those

**Comment by murtaugh** [comment]
Sorry, I wasn't very clear:

I need a way to filter between entries that have hit their entry date and those that have not. 

No matter what date filter I select, I get all future posts as well as entries that match the date filter, i.e., if I select "Last 24 Hours" I get all future entries along with all entries from the last 24 hours.

In my particular case, I'm working with a site that manages a lot of events, and there's no way to see "Last 24 Hours" without wading through all future events first.

I think all the filters should exclude future dates, since that's what their description implies, and there should be a new filter for entries dated in the future.

**Comment by intoeetive** [comment]
https://github.com/ExpressionEngine/ExpressionEngine/pull/4806
